### DISCUSSION TITLE
Is there a way to force a `:hover` / `:active` state with the react-aria Button?

### DISCUSSION THREAD

**Discussion by sbdchd** [adobe/react-spectrum#5667] [Q&A] [answered]
My use case is a little niche, but essentially want to have a storybook like setup where I display all the button variants and their possible states so I can better adjust their colors

I was hoping I could use `data-hovered` and co style the components and maybe force those to be set via a prop?

Not too tricky to work around, but curious about this sort of use case!

**Comment by reidbarber** [comment, marked as answer]
Have not tested, but would this addon work? https://github.com/chromaui/storybook-addon-pseudo-states

Or maybe interaction tests with play functions: https://storybook.js.org/docs/writing-tests/interaction-testing

**Reply by jared-christensen** [reply]
The add-on did not work for me.The play functions work but not really what I'm looking for.
